extensions/wikia/PageShare/scripts/PageShare.js

```diff
@@ -58,6 +59,10 @@ require(['wikia.window', 'wikia.tracker', 'jquery'], function(win, tracker, $) {
 			requestData.mcache = mCache;
 		}
 
+		if (useLang) {
+			requestData.useLang = useLang;
+		}
+
 		$.nirvana.sendRequest({
```

Comment:
Did you consider doing a `jsonp` request to `http://api.wikia.nocookie.net` domain?